CMakeLists.txt

```diff
@@ -281,45 +276,11 @@ set(CMAKE_INSTALL_NAME_DIR
 # enable 24 bit receiving path
 if (RX_SAMPLE_24BIT)
     message(STATUS "Compiling for 24 bit Rx DSP chain")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DSDR_RX_SAMPLE_24BIT")
+    add_compile_definitions(SDR_RX_SAMPLE_24BIT)
 else()
     message(STATUS "Compiling for 16 bit Rx DSP chain")
 endif()
 
-if (SANITIZE_ADDRESS)
-    message(STATUS "Activate address sanitization")
-    if(MSVC)
-      set(ASAN_LIB_ARCH ${MSVC_CXX_ARCHITECTURE_ID})
-      string(TOLOWER ${ASAN_LIB_ARCH} ASAN_LIB_ARCH)
-      if(ASAN_LIB_ARCH STREQUAL "x86")
-        set(ASAN_LIB_ARCH "i386")
-      elseif(ASAN_LIB_ARCH STREQUAL "x64")
-        set(ASAN_LIB_ARCH "x86_64")
-      endif()
-      add_compile_options(/fsanitize=address)
-      link_libraries(clang_rt.asan_dynamic-${ASAN_LIB_ARCH} clang_rt.asan_dynamic_runtime_thunk-${ASAN_LIB_ARCH})
-      add_link_options(/wholearchive:clang_rt.asan_dynamic_runtime_thunk-${ASAN_LIB_ARCH}.lib)
-    else()
-      add_compile_options(-fsanitize=address -fno-omit-frame-pointer -g)
-      add_link_options(-fsanitize=address)
-    endif()
-endif()
-
-# set compiler
-include(FindCompiler)
-
-if (C_CLANG OR C_GCC)
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wvla -Woverloaded-virtual -ffast-math -ftree-vectorize ${EXTRA_FLAGS}")
```

Comment:
Why remove this?

Reply:
Not removed, moved to find flags cmake. `FindCompiler` was duplicated there and I stared this patch before you actually fixed this warnings. But same flags are set https://github.com/f4exb/sdrangel/pull/714/files#diff-1b53431f5e291976688daa17a2a78f1c0d4ca5bc0cd5559f9aed23844ba2789fR217

Reply:
It is still not clear where these flags are set now.

Reply:
Ok, I will change it to make more sense.